src/services/coreBackService.js

```diff
@@ -0,0 +1,83 @@
+const {
+  API_BASE_URL,
+  API_BUILD_PROVEN_USER_IDENTITY_DETAILS,
+  API_CRI_CALLBACK,
+  API_SESSION_INITIALISE
+} = require("../lib/config");
+const https = require("https");
+const axios = require("axios");
+
+class CoreBackService {
```

Comment:
Does this need to be a class - I think we could just make this a module? (i.e. just export `postSessionInitialise`, `postCriCallback` and `getProvenIdentityDetails` directly)

The only instance variable is the axios instance, and I think we _do_ want to share it between different consumers. Plus it'll probably make jest mocking easier.